media/css/mozorg/home/home-promo.less

```diff
@@ -1066,6 +1066,54 @@ html[lang|="en"] {
 
 
 /****************************************************************************/
+// End of year fundraising
+.promo-large-landscape.fundraiser {
+    background-color: #e63d2f;
+    color: #fff;
+
+    h2 {
+        padding-top: 30px;
+        .font-size(32px);
+    }
+
+    h2:before {
+        display: block;
+        content: '';
+        background: transparent url('/media/img/home/voices/promos/fundraising/fox.svg') 0 0 no-repeat;
+        background-size: 134px 115px;
+        margin: 0 auto;
+        width: 134px;
+        height: 115px;
+    }
+
+    h3 {
+        padding-top: 90px;
+    }
+
+    &.show,
+    &.show-touch {
```

Comment:
This `.show-touch` doesn't have enough specificity to apply the border override, since the original rule also has `.promo-grid.has-touch` as a parent.